src/main/webapp/js/projects/react/prop_types.js

```diff
@@ -1,3 +1,6 @@
+import {checkTesting} from '../../checkTesting.js';
+checkTesting();
```

Comment:
nit: consider changing `checkTesting` to something that is more descriptive of what this function is doing. IMO something like `setupIfTesting`, `configureGlobalsForEnvironment` or `setGlobalsIfTesting` would be better.

Reply:
I'd expect that a function called `checkTesting` would return a boolean telling whether this was a test environment or not. I wouldn't expect it to modify the global state.

Reply:
I totally agree. I'll change it to `setGlobalsIfTesting`.